README.md

```diff
@@ -79,3 +79,42 @@ The following examples are provided:
 * [02 - Coloured Triangle](src/example_02/README.md)
 * [03 - Using an Index Buffer](src/example_03/README.md)
 * [04 - Rotating Cube](src/example_04/README.md)
+
+## Doc Site
+The documentation site is served by GitHub pages. The generate is Jekyll. There
```

Comment:
Eventually shouldn't we do that in https://github.com/webgpu-native/webgpu-headers ?

Reply:
Possibly? I'm happy to move things over if we like how this ends up.